int8 question
---
int8 answer
